modules/phase_field/src/base/PhaseFieldApp.C

```diff
@@ -488,7 +492,8 @@ PhaseFieldApp::associateSyntax(Syntax & syntax, ActionFactory & action_factory)
   registerAction(EulerAngle2RGBAction, "add_aux_kernel");
   registerAction(HHPFCRFFSplitKernelAction, "add_kernel");
   registerAction(HHPFCRFFSplitVariablesAction, "add_variable");
-  registerAction(MatVecRealGradAuxKernelAction, "add_aux_kernel");
```

Comment:
Ok, but then you should not change/remove its functionality in the class source.